src/cfnlint/helpers.py

```diff
@@ -171,6 +171,39 @@
     'AWS::Redshift::Cluster'
 ]
 
+VALID_PARAMETER_TYPES_SINGLE = [
+    'AWS::EC2::AvailabilityZone::Name',
+    'AWS::EC2::Image::Id',
+    'AWS::EC2::Instance::Id',
+    'AWS::EC2::KeyPair::KeyName',
+    'AWS::EC2::SecurityGroup::GroupName',
+    'AWS::EC2::SecurityGroup::Id',
+    'AWS::EC2::Subnet::Id',
+    'AWS::EC2::VPC::Id',
+    'AWS::EC2::Volume::Id',
+    'AWS::Route53::HostedZone::Id',
+    'AWS::SSM::Parameter::Name',
```

Comment:
what about the other documented [`AWS::SSM::Parameter` parameter types](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html#aws-ssm-parameter-types)?

Reply:
good call.